sdk/test-utils/recorder-new/MIGRATION.md

```diff
@@ -0,0 +1,245 @@
+# Migration Guide
+
+This document outlines key differences between the legacy recorder and the new Unified Recorder client. The Unified Recorder replaces the existing `nock/nise`-based recorder with a solution that uses the language-agnostic [test proxy server].
+
+## Prerequisites
+
+- [Docker] is required, as the [test proxy server] is run in a container during testing.
+
+## Installing the Unified Recorder
+
+The first step is to install the unified recorder client library in your package by running
+
+```bash
+$ rush add --dev -p @azure-tools/test-recorder-new
```

Comment:
Running this adds the following dependency `"@azure-tools/test-recorder-new": "~1.0.0"` note the `~`. We should add it as a caret dependency?

Reply:
yes, caret should be good

Reply:
Oddly enough, my project doesn't resolve `"@azure-tools/test-recorder-new"`.
I added `"@azure-tools/test-recorder-new": "^1.0.0"` to my dev-dependencies and after `rush update` there are no changes in my lock file

Reply:
Can you post your branch?

Reply:
Probably some messy pnpm-lock file or build outputs?